e2ev2/tests/user-create.test.js

```diff
@@ -0,0 +1,94 @@
+const config = require('../src/config')
+const tu = require('../src/test-utils')
+const u = require('../src/utils')
+
+const browserTest = tu.browserTest(__filename)
+
+// Puppeteer's Page.click is not reliable. This is worth investigation.
+const jsClick = (page, sel) => page.waitForSelector(sel).then(eh => eh.evaluate(e => e.click()))
+
+const pressKey = async (page, keyName, count = 1) => {
+  for (let i = 0; i < count; ++i) {
+    await page.keyboard.press(keyName)
+  }
+}
+
+browserTest('create user', async browser => {
+  const page = browser.initialPage
+  const iframeSel = '[aria-label="terms of use and privacy statement"]>iframe'
+  const ackCheckboxSel = '[data-test-id="agreement-check"]'
+  const isAckCheckboxDisabled =
+    () => page.waitForSelector(ackCheckboxSel).then(eh => eh.evaluate(e => e.disabled))
+
+  await tu.useApiProxy(page)
+  await page.goto(config.urlRoot(), {waitUntil: 'networkidle0'})
+  await jsClick(page, '[data-test-id="login"]>div+div>[role="button"]') // Create Account
+
+  await page.waitForSelector(iframeSel) // Agreement Page
+  await page.evaluate(() => window.forceHasReadEntireAgreement())
+  await expect(isAckCheckboxDisabled()).resolves.toBe(false)
+  await jsClick(page, ackCheckboxSel)
+  await page.click('[role="button"]') // Next button
+
+  await page.waitForSelector('.p-dropdown') // Institution Page
```

Comment:
I'm seeing a lot of `waitForSelector()` then `click()` with the same arg, so I'm inclined to add a helper function.

Reply:
Oh good point. I'm going to merge this as-is to avoid some churn, but that's the first thing I'll look at in my follow-up branch. I want to give it some thought, since the wait and click are kind of two different things.